.gitignore

```diff
@@ -66,6 +66,9 @@ anaconda-mode/
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+# Ignore leftovers created by `hack/` scripts
+_tmp/
+vendordiff.patch
```

Comment:
Given you are adding this, there is also a `deploy/test/` folder that can be added here.

Reply:
Done